app/javascript/components/widgets/numbers/panel.jsx

```diff
@@ -57,15 +76,34 @@ class SubscribedEvents extends React.Component {
         <NumberWrapper>
           <div>
             <Count>{githubCommitCount}</Count>
-            <CountLabel color={colors.pink}>{commitText}</CountLabel>
+            <IconTextWrapper>
+              <BlockIcon
+                src={githubCommitImage}
+                width="24"
+                height="24"
+                alt=""
+              />
+              <CountLabel color={colors.pink}>{commitText}</CountLabel>
+            </IconTextWrapper>
           </div>
           <div>
             <Count>{githubPullCount}</Count>
-            <CountLabel color={colors.yellow}>Pull {requestText}</CountLabel>
+            <IconTextWrapper>
+              <BlockIcon src={githubPullImage} width="24" height="24" alt="" />
+              <CountLabel color={colors.yellow}>Pull {requestText}</CountLabel>
+            </IconTextWrapper>
           </div>
           <div>
             <Count>{slackMessageCount}</Count>
-            <CountLabel color={colors.teal}>Slack {messageText}</CountLabel>
+            <IconTextWrapper>
+              <BlockIcon
+                src={slackMessageImage}
+                width="24"
+                height="24"
+                alt=""
+              />
+              <CountLabel color={colors.teal}>Slack {messageText}</CountLabel>
```

Comment:
Have you considered using .map on this?

Reply:
I think extracting the duplicated code a named component than a map of props. The goal being:
```
<React.Fragment>
   <CountSummary count={} image={} textColor={} text={} />
   <CountSummary count={} image={} textColor={} text={} />
   <CountSummary count={} image={} textColor={} text={} />
</React.Fragment>
```